src/190B20.c

```diff
@@ -3502,7 +3502,7 @@ void btl_update_ko_status(void) {
     player->koDuration = player->debuffDuration;
     if (player->koDuration > 0) {
         player->koStatus = STATUS_DAZE;
-        ((s32*)player->debuffIcon->data)[15] = player->koDuration;
+        ((DisableXFXData*)player->debuffEffect->data)->unk_3C = player->koDuration;
```

Comment:
We can totally name this `unk_3C` field `koDuration`

Reply:
it's definitely some sort of duration, but there's no indication that it's specifically a ko duration just because that's used for it in this instance

Reply:
clover was gonna work through these some point soon anyway, btw